src/pages/postgraphile/running-postgraphile-in-docker.md

```diff
@@ -0,0 +1,598 @@
+---
+layout: page
+path: /postgraphile/running-postgraphile-in-docker/
+title: Running PostGraphile in Docker
+---
+
+The following guide describes how to run a network of Docker containers on a local machine, including one container for a PostgreSQL database and one container for PostGraphile. A the end of this guide, you will have a GraphQL API exposing data from a PostgreSQL database, both running locally on your machine in separate Docker containers. It has been developed and tested on:
+
+-   Linux
+-   Windows Pro
+-   Windows Home
+
+# Table of Contents
+
+-   [Requirements](#requirements)
+    -   [Install Docker and Docker Compose on Linux](#install-docker-and-docker-compose-on-linux)
+    -   [Install Docker on Windows Pro](#install-docker-on-windows-pro)
+    -   [Install Docker on Windows Home](#install-docker-on-windows-home)
+-   [Create PostgreSQL Container](#create-postgresql-container)
+    -   [Setup Environment Variables](#setup-environment-variables)
+    -   [Create Database Initialization Files](#create-database-initialization-files)
+    -   [Create PostgreSQL Dockerfile](#create-postgresql-dockerfile)
+    -   [Create Docker Compose File](#create-docker-compose-file)
+-   [Create PostGraphile Container](#create-postgraphile-container)
+    -   [Update Environment Variables](#update-environment-variables)
+    -   [Create PostGraphile Dockerfile](#create-postgraphile-dockerfile)
+    -   [Update Docker Compose File](#update-docker-compose-file)
+-   [Build Images And Run Containers](#build-images-and-run-containers)
+    -   [Build Images](#build-images)
+    -   [Run Containers](#run-containers)
+    -   [Re-initialize The Database](#re-initialize-the-database)
+-   [Add Custom Plugin](#add-custom-plugin)
+    -   [makeWrapResolversPlugin](#makewrapresolversplugin)
+-   [Queries And Mutations Examples](#queries-and-mutations-examples)
+    -   [Queries](#queries)
+    -   [Mutations](#mutations)
+
+# Requirements

```

Comment:
I should really fix our markdown generation; but for now H1 and H2 are reserved, so you need to prefix all headers with two more hashes, e.g.:

```suggestion
### Requirements
```

Reply:
Ok, doing it.